import-dashboards.py

```diff
@@ -19,16 +19,18 @@
 
 import requests
 
-GRAFANA_DB_DIR   = sys.argv[1] if len(sys.argv) > 1 else '/srv/grafana'
-GRAFANA_IMG_DR   = '/usr/share/grafana/public/img/'
-SCRIPT_DIR       = os.path.dirname(os.path.abspath(__file__))
-DASHBOARD_DIR    = SCRIPT_DIR + '/dashboards/'
-NEW_VERSION_FILE = SCRIPT_DIR + '/VERSION'
-OLD_VERSION_FILE = GRAFANA_DB_DIR + '/PERCONA_DASHBOARDS_VERSION'
-HOST             = 'http://127.0.0.1:3000'
-LOGO_FILE        = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
-SET_OF_TAGS 	 = {'QAN': 0, 'OS': 0, 'MySQL': 0, 'MongoDB': 0, 'PostgreSQL': 0, 'HA': 0, 'Cloud': 0, 'Insight': 0, 'PMM': 0}
-CONTENT          = '''<center>
+GRAFANA_DB_DIR        = sys.argv[1] if len(sys.argv) > 1 else '/srv/grafana'
+GRAFANA_IMG_DR        = '/usr/share/grafana/public/img/'
+SCRIPT_DIR            = os.path.dirname(os.path.abspath(__file__))
+DASHBOARD_DIR         = SCRIPT_DIR + '/dashboards/'
+NEW_VERSION_FILE      = SCRIPT_DIR + '/VERSION'
+OLD_VERSION_FILE      = GRAFANA_DB_DIR + '/PERCONA_DASHBOARDS_VERSION'
+GRAFANA_PROVISION_DIR = '/usr/share/grafana/conf/provisioning/dashboards/'
+PMM_PLUGIN_DIR        = '/var/lib/grafana/plugins/pmm-app/dist/dashboards/'
+HOST                  = 'http://127.0.0.1:3000'
+LOGO_FILE             = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
+SET_OF_TAGS 	      = {'QAN': 0, 'OS': 0, 'MySQL': 0, 'MongoDB': 0, 'PostgreSQL': 0, 'HA': 0, 'Cloud': 0, 'Insight': 0, 'PMM': 0}
+CONTENT               = '''<center>
```

Comment:
Such align is dis-recommended by [PEP 8/Pet Peeves](https://www.python.org/dev/peps/pep-0008/#pet-peevesl)

> Avoid extraneous whitespace in the following situations:
>   More than one space around an assignment (or other) operator to align it with another.

Reply:
Fixed